sdk/core/azure-core-http-netty/pom.xml

```diff
@@ -116,7 +116,14 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.2.0-beta.1</version>  <!-- {x-version-update;com.azure:azure-core-test;current} -->
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-test;current} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-test</artifactId>
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-test;current} -->
+      <type>test-jar</type>
```

Comment:
If you have the test-jar, do you need the one without the test-jar too?

Reply:
I'll try this out locally, if it continues to work the same I'll remove the other.

Reply:
Build failed